### PR TITLE
[OSAC-367] - [Cryptography]Expose output filename to ViewModel

### PR DESCRIPTION
**Pull request by nicolae-anghel**
https://uipath.atlassian.net/browse/OSAC-367

### CODE REVIEW

File: Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx [hunk 1]
```diff
@@ -312,7 +312,7 @@
     <comment>Property description</comment>
   </data>
   <data name="Activity_EncryptFile_Property_Algorithm_Name" xml:space="preserve">
-    <value>Algortithm</value>
+    <value>Algorithm</value>
```
Comment (alcxs) [inline Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx:315]: Thanks

File: Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs [hunk 2]
```diff
@@ -64,6 +64,11 @@ public EncryptFile()
         [LocalizedDescription(nameof(Resources.Activity_EncryptFile_Property_OutputFilePath_Description))]
         public InArgument<string> OutputFilePath { get; set; }
 
+        [LocalizedCategory(nameof(Resources.Input))]
+        [LocalizedDisplayName(nameof(Resources.Activity_EncryptFile_Property_OutputFileName_Name))]
+        [LocalizedDescription(nameof(Resources.Activity_EncryptFile_Property_OutputFileName_Description))]
+        public InArgument<string> OutputFileName { get; set; }
```
Comment (alcxs) [inline Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs:70]: please add similar to Decrypt File as well

**Review by alcxs — CHANGES_REQUESTED** [review]
do the same for Decrypt File as well

File: Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx (hunk 1 above)
Comment (nicolae-anghel) [reply Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx:315]: You are welcome ;)

File: Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs (hunk 2 above)
Comment (nicolae-anghel) [reply Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs:70]: Added

**Review by alcxs — APPROVED** [review]

File: Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx (hunk 1 above)
Comment (ovidiuiliescu) [inline Activities/Cryptography/UiPath.Cryptography.Activities/Properties/UiPath.Cryptography.Activities.resx:315]: 👍

**Review by ovidiuiliescu — APPROVED** [review]

**Pull request merged**